modules/fs2/src/smithy4s/interopfs2/package.scala

```diff
@@ -16,14 +16,5 @@
 
 package smithy4s
 
-import fs2.Stream
-
 // scalafmt: {maxColumn = 120}
-package object interopfs2 {
-
-  type ByteStreamEncoder[F[_], A] =
-    smithy4s.codecs.Encoder[Stream[F, Byte], A]
-  type ByteStreamDecoder[F[_], A] =
-    smithy4s.codecs.Decoder[F, Stream[F, Byte], A]
-
-}
+package object interopfs2 {}
```

Comment:
Is this just being left as a placeholder for future use?

Reply:
yes